AugurProject/augur
# fixes and some cleaning up of claiming proceeds.

error with position summary being null after claiming which shouldn't be the case. positions go to closed.  add a little bit of clean up for clarity and maintainability.

`MODAL_CLAIM_TRADING_PROCEEDS` was removed `MODAL_CLAIM_MARKETS_PROCEEDS` was being used in only a few places. convert to only use `MODAL_CLAIM_MARKETS_PROCEEDS`

to test:

push market to awaiting finalization
create some orders
get winning shares
claim proceeds and verify no error in web console.
